doc/operator-notes.md

```diff
@@ -30,3 +30,34 @@ In the second two cases, where there is a preexisting filesystem, Ignition's beh
 If `wipeFilesystem` is set to true, Ignition will always wipe any preexisting filesystem and create the desired filesystem. Note this will result in any data on the old filesystem being lost.
 
 If `wipeFilesystem` is set to false, Ignition will then attempt to reuse the existing filesystem. If the filesystem is of the correct type, has a matching label, and has a matching UUID, then Ignition will reuse the filesystem. If the label or UUID is not set in the Ignition config, they don't need to match for Ignition to reuse the filesystem. Any preexisting data will be left on the device and will be available to the installation. If the preexisting filesystem is *not* of the correct type, then Ignition will fail, and the machine will fail to boot.
+
+## Partition Reuse Semantics
+Similar to the filesystemd use semantics, there are three possibilties for partitions when Ignition runs
+
+- The is no preexisting partition
+- There is a preexisting partition of the correct size, starting sector, label, type_guid and guid.
+- There is a preexisting partition of an incorrect size, starting sector, label, type_guid or guid.
+
+In the first case, Ignition will always create the desired partition. Using a partition number of 0 specifies to use the next available partition number and will ensure there is no preexisting partition.
+
+In the second two cases, the wipePartition flag controls Ignition's behavior. If it is set to true, Ignition will first delete the partition with the specified number and then attempt to create the new partition. The partition number must be specified and non-zero if wipePartition is set to true.
+
+If wipePartition is false, Ignition will attempt to reuse the existing partition. If the partition has a matching label, uuid, type_guid, start sector, and size, Ignition will reuse the partition. If any of those are not specified, zero, or empty, they do not need to match for Ignition to reuse the partition. If the partition does not match the Ignition configuration, Ignition will fail, and the machine will fail to boot.
+
+## Raid Reuse Semantics
+Just like filesystems and partitions, there are three possibilities for raid arrays when Ignition runs
+
+- The devices specified by the RAID array are unused
+- The devices specified by the RAID array are used in an existing RAID array with the correct level, devices, and name
+- The devices specified by the RAID array are used are either not a member of an existing array or are a member of an exisiting RAID array with incorrect level, devices, or name
+
+An unused device is
+ * Not part of an exising RAID array
+ * A partition with no filesystem present
```

Comment:
(or other metadata, such as an LVM PV)

Reply:
Do we want to check for things we don't support creating? It seems like we would need a lot of checks for misc types, unless you know a utility which will catch most of them.

Reply:
blkid is what I was thinking of.  We should try to avoid clobbering things even if we don't know how to create them.